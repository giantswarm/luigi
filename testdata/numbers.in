{"caller":123,"level":"error","time":"2019-09-06T12:40:27.581957+00:00","verbosity":0}
