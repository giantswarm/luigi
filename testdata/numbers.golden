E 09/06 12:40:27 | 123 | verbosity=0
